config/payment.php

```diff
@@ -84,7 +86,7 @@
             'apiPurchaseUrl' => 'https://pay.ir/pg/send/',
             'apiPaymentUrl' => 'https://pay.ir/pg/',
             'apiVerificationUrl' => 'https://pay.ir/pg/verify/',
-            'merchantId' => '', // set it to `test` for test environments
+            'merchantId' => 'test', // set it to `test` for test environments
```

Comment:
please let it to be empty! its default value must not be test. it must be empty, so it forces users to have a look at config file and set merchanid.